tools/launch_benchmark_suite_scheduler_slurm.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+import submitit
+from fvcore.common.file_io import PathManager
+from vissl.config.attr_dict import AttrDict
+from vissl.engines.benchmark_suite_scheduler import BenchmarkSuiteScheduler
+from vissl.utils.io import load_file
+
+
+# Default slurm options to pass to the executor.
+_DEFAULT_SLURM_OPTIONS = {
+    "NAME": "vissl",
+    "COMMENT": "vissl evaluation job",
+    "PARTITION": "learnlab",
```

Comment:
nit: this is specific to FAIR  cluster. we should remove this from defaults. :)

Reply:
Should we also remove "PARTITION: "learnfair"" From defaults.yaml?

Reply:
that's fine with me :)